src/modules/slack/slackHandlers.ts

```diff
@@ -63,8 +63,9 @@ export const towerSlackActionHandler: Lifecycle.Method = async (request, _h) =>
   const slackActionPayload = request.pre.slackActionPayload;
   try {
     switch (slackActionPayload.type) {
-      case 'shortcut':
-      // return handleTowerShortcut(slackActionPayload as SlackShortcutPayload);
+      // case 'shortcut':
+      // // return handleTowerShortcut(slackActionPayload as SlackShortcutPayload);
+      //   break
```

Comment:
need confirmation if we can remove case: 'shortcut' here. @ccmoralesj 
Since there was no break, the current code would call handleTowerAction() (line 70) in case the slackActionPayload.type === 'shortcut' . What is the expected behavior?

Reply:
Slack has 2 main ways to call APIs from the chat. 

1. Call them with Slack commands /something
2. Call them with Action. Buttons inside the plus (+) symbol in the left corner of the chat

Both work the same way internally so we had a case without a break because we will call the same function in both cases. I think we can safely remove that commented line and leave the `case 'shortcut'` as it is because Slack will continue sending one or the other but we handle them the same way.

Reply:
done. uncommented case: shortcut